lib/awspec/helper/finder/cloudwatch_logs.rb

```diff
@@ -0,0 +1,27 @@
+module Awspec::Helper
+  module Finder
+    module CloudwatchLogs
+      def find_cloudwatch_logs_group(id)
+        cloudwatch_logs_client.describe_log_groups({ log_group_name_prefix: id }).log_groups.last
+      end
+
+      def find_cloudwatch_logs_stream(id)
```

Comment:
`find_cloudwatch_logs_stream_by_log_group_name` ?
( This method get stream with `log_group_name` instead of `log_stream_name_prefix` )